internal/utils/eszip/build.ts

```diff
@@ -21,6 +21,12 @@ async function buildAndWrite(p: string) {
       if (specifier.startsWith(virtualBasePath)) {
         actualPath = specifier.replace(virtualBasePath, `./${funcDirPath}/`);
       }
+
+      // If an import map path is set read file from the given path.
+      // Otherwise default to `import_map.json` in functions directory.
+      if (specifier.endsWith('import_map.json') && importMapPath != "") {
```

Comment:
```suggestion
      if (specifier.endsWith('import_map.json') && importMapPath) {
```

`Deno.args[1]` may be undefined if called with only 1 arg. Safer to handle both cases